tests/test_datasets/test_dataset.py

```diff
@@ -284,9 +287,14 @@ def test_get_sparse_dataset(self):
         self.assertEqual(len(attribute_names), 20001)
         self.assertTrue(all([isinstance(att, str) for att in attribute_names]))
 
+    @unittest.skipIf(LooseVersion(pd.__version__) < "1.0.0",
+                     reason="SparseDataFrame support removed from pandas 1.0.0 and onwards.")
     def test_get_sparse_dataframe(self):
         rval, *_ = self.sparse_dataset.get_data()
-        self.assertTrue(isinstance(rval, pd.SparseDataFrame))
+        dtypes = all([dtype == pd.SparseDtype(np.float32, fill_value=0.0)
+                      for dtype in rval.dtypes])
+        self.assertTrue(isinstance(rval, pd.DataFrame))
+        self.assertTrue(dtypes)
```

Comment:
Could you please use `np.testing.assertAllEqual` here?

Reply:
I couldn't find that function but I am using `np.testing.assert_array_equal` now